sycl/test-e2e/lit.cfg.py

```diff
@@ -253,6 +253,8 @@
     devices = set()
     sp = subprocess.check_output(sycl_ls, text=True)
     for line in sp.splitlines():
+        if "gfx90a" in line:
+            config.available_features.add("gpu-amd-gfx90a")
```

Comment:
I'm ok with it here in this PR, but if we ever extend with one more device, I'd like to have more structure in such auto-detection code.